README.md

```diff
@@ -20,3 +20,37 @@ This project was started in Fall 2019 and is currently in development.
 * Ritika Rao
 * Megan Niu - [@meganniu](https://github.com/meganniu)
 * Jayant Shrivastava - [@jayshrivastava](https://github.com/jayshrivastava)
+
+## Development Setup
+
+**Development DB Setup:** 
```

Comment:
Is there a way that we can abstract the docker logic into simple scripts that we can call?

Reply:
Probably. I would say leave it for now for two reasons:
1. There's only 3 lines (pull, start a container, rake drop+create+migrate+seed)
2. You only need to do the first two once. After that, `rake db:<command>` can be used. If we put all of them into one command, one would be inclined to use the command every time they need to reset the db. This would mean spinning up a container every time without shutting down the previous one. I think it's better for devs to understand what the docker commands do and leave them separate.